src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -203,6 +203,13 @@ def get_object_info3(self, params):
     def run_job(self, params):
         return self.test_job_id
 
+    def run_job_batch(self, batch_job_inputs, batch_params):
```

Comment:
This (and other functions here) are super simple dummy mocks that just return happy meaningless results. In this case, one job id for each child job, and one for the parent.